python/ray/tests/BUILD

```diff
@@ -65,6 +61,18 @@ py_test_module_list(
   deps = ["//:ray_lib", ":conftest"],
 )
 
+py_test_module_list(
+  files = [
+    "test_gcs_fault_tolerance.py",
+    "test_global_gc.py",
+    "test_job.py",
+    "test_memstat.py",
+  ],
+  size = "medium",
+  tags = ["exclusive", "medium_size_python_tests_a_to_j", "no_windows", "team:core"],
+  deps = ["//:ray_lib", ":conftest"],
+)
```

Comment:
why are these no windows? was this defined in the no windows list in the `ci/ci.sh`?

Reply:
yes, they are from this list https://github.com/ray-project/ray/blob/master/ci/ci.sh#L224